tests/expected/function-contract/modifies/mistake_condition_return.expected

```diff
@@ -0,0 +1,4 @@
+Failed Checks: assertion failed: res == 100 \
+ File: "mistake_condition_return.rs", line 31, in use_modify \
```

Comment:
please don't add the entire file path neither the exact line since they are painful to maintain. The file path changes depending where you are running the tests from. The line number forces us to change this expected file even if just a comment is added to the original test.